File: ChatServerVB/IClient.vb
Public Interface IClient
    Sub ParticipantDisconnection(name As String)
    Sub ParticipantReconnection(name As String)
    Sub ParticipantLogin(client As User)
    Sub ParticipantLogout(name As String)
    Sub BroadcastTextMessage(sender As String, message As String)
    Sub BroadcastPictureMessage(sender As String, ByVal img As Byte())
    Sub UnicastTextMessage(sender As String, message As String)
    Sub UnicastPictureMessage(sender As String, ByVal img As Byte())
    Sub ParticipantTyping(sender As String)
End Interface

File: ChatServerCS/IClient.cs
namespace ChatServerCS
{
    public interface IClient
    {
        void ParticipantDisconnection(string name);
        void ParticipantReconnection(string name);
        void ParticipantLogin(User client);
        void ParticipantLogout(string name);
        void BroadcastTextMessage(string sender, string message);
        void BroadcastPictureMessage(string sender, byte[] img);
        void UnicastTextMessage(string sender, string message);
        void UnicastPictureMessage(string sender, byte[] img);
        void ParticipantTyping(string sender);
    }
}